tests/unit/lms/views/content_item_selection_test.py

```diff
@@ -108,10 +44,3 @@ def context(self):
 
```

Comment:
Can't really comment the right line, but the `routes` fixture here doesn't appear to make any difference any more and can be removed.

The `custom_canvas_api_domain` and `custom_canvas_course_id` also don't appear to have a bearing on this test.

Reply:
Nope, well spotted